guava/src/main/java/com/github/benmanes/caffeine/guava/CaffeinatedGuavaLoadingCache.java

```diff
@@ -123,7 +123,7 @@ public V apply(K key) {
   }
 
   @Override
-  @SuppressWarnings({"CheckReturnValue", "FutureReturnValueIgnored"})
+  @SuppressWarnings("FutureReturnValueIgnored")
```

Comment:
We generally have been recommending just capturing into a `var unused = ...` variable instead of using `@SuppressWarnings(...)`. Up to you tho.

Reply:
I see 36 usages of `@SuppressWarnings("FutureReturnValueIgnored")`. Happy to make this change, but perhaps this is better done in a separate PR. Will await @ben-manes' verdict :)

Reply:
I'd prefer `@SuppressWarnings` as other linters will flag unused variables (e.g. Eclipse). It's unfortunate as it does expand the suppression to the method instead of statement, but does a better job of avoiding warning spam. I'm open to revisiting to a better scheme separately so as to narrow the scope.